### DISCUSSION TITLE
Very slow startup

### DISCUSSION THREAD

**Discussion by Konglomneshued** [wezterm/wezterm#4267] [Q&A]
WezTerm appears to take almost two seconds to start.  What can I do to make this more responsive?

I installed through the AUR both the normal and the git version and they both start the same speed.

I took a look at similar discussions with no luck.

Timings:
```bash
real    0m0.101s
user    0m0.062s
sys     0m0.030s
```

**Comment by wez** [comment]
It's impossible to give you decent advice because you didn't share any useful information.

Search for other issues to see if any sound similar to your environment.
At a guess, there's an issue with your graphics drivers. look for issues that mention xrandr.
Alternatively, there's an issue with font-config and you need to rebuild its cache.

If you want to pursue this further, please use the issue template and gather all of the information it requests.
